TestProviders/HyperVProvider.psm1

```diff
@@ -157,6 +157,8 @@ Class HyperVProvider : TestProvider
 				}
 			}
 
+			$allVMData = Get-AllHyperVDeployementData -HyperVGroupNames $allVMData[0].HyperVGroupName -CurrentTestData $CurrentTestData
```

Comment:
It's better to split to two commits. One is for path, another one is for refresh IPs.

Reply:
They can be two commits, but in the same PR. It depends on you.

Reply:
Sorry, I open another PR #1148